image.yaml

```diff
@@ -53,7 +53,7 @@ modules:
           - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
           - name: jboss.eap.cd16.openshift
-          - name: jboss.eap.cd.openshift.modules
+#          - name: jboss.eap.cd.openshift.modules
```

Comment:
I don't think this can be removed.

1) The cekit module installs a JBoss Modules module used by tx recovery.
2) The user can bring their own standalone.xml and it can point to the legacy openshift.PING stuff; i.e. just because our scripts won't use those doesn't a user won't.  We could drop the modules anyway. Perhaps in 17?

Reply:
OK, I can add this back. In 16 AFAIR, we marked them as deprecated in the docs, so if a user did bring their own config that had them, we could just grep for them and warn.

I'll double check the tx recovery one, I thought it was just the clustering alias one that was left.

Reply:
Never mind, I'd commented out the completely wrong module, fixing...

Reply:
@bstansberry updated and whole topic branch squashed.,

Reply:
FYI, this was removed as the pings modules were dropped, but as you pointed out, they're needed if someone brings their own configs, so I've restored both. +1 to maybe in 17.